src/index.html

```diff
@@ -1,5 +1,6 @@
 <!doctype html>
 <html>
+  <script type="text/javascript" src="/assets/appInsights.js"></script>
```

Comment:
According to the appinsights docs you pasted earlier, this should go before `</head> `not before `<head>`

Reply:
Application insights works correctly by injecting the codes here. So it should be fine leaving it this way.